### PR TITLE
[MSYS-645] windows_path resource docs

### PR DESCRIPTION
**Pull request by NimishaS**
w.r.t. https://github.com/chef/chef/pull/6295

### CODE REVIEW

**Comment by NimishaS** [comment]
@btm, please review.

File: chef_master/resource_windows_path.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
+==========================================
+windows_task
+==========================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/resource_windows_path.rst>`__
+
+Use the **windows_path** resource to manage path environment key in Microsoft Windows.
```
Comment (btm) [inline chef_master/resource_windows_path.rst]: "to manage the path environment variable on Microsoft Windows."

File: chef_master/resource_windows_path.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+==========================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/resource_windows_path.rst>`__
+
+Use the **windows_path** resource to manage path environment key in Microsoft Windows.
+
+*New in Chef Client 13.*
+
+Syntax
+==========================================
+A **windows_path** resource manages path environment key in Microsoft Windows.
```
Comment (btm) [inline chef_master/resource_windows_path.rst]: ... manages the path environment variable on Microsoft Windows.

**Review by btm — APPROVED** [review]

File: chef_master/resource_windows_path.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
+==========================================
+windows_task
+==========================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/resource_windows_path.rst>`__
+
+Use the **windows_path** resource to manage the path environment variable on Microsoft Windows.
+
+*New in Chef Client 13.*
```
Comment (tas50) [inline chef_master/resource_windows_path.rst]: This should be 13.4

**Pull request merged**